Packs/VMware/Integrations/VMware/VMware.yml

```
@@ -208,11 +259,54 @@ script:
     - description: VM UUID of virtual machine to get events of.
       name: vm-uuid
       required: true
-    - defaultValue: VmGuestRebootEvent,VmGuestShutdownEvent,VmPoweredOnEvent,VmPoweredOffEvent,VmSuspendedEvent
+    - name: event-type
       description: Type of events to get, given in CSV (i.e.,  VmGuestRebootEvent,VmGuestShutdownEvent).
-      name: event-type
+      required: false
+      isArray: false
+      defaultValue: VmGuestRebootEvent,VmGuestShutdownEvent,VmPoweredOnEvent,VmPoweredOffEvent,VmSuspendedEvent
+    - name: start-date
+      description: Event's start date.
+      required: false
+      isArray: false
+    - name: end-date
+      description: Event's end date
```

our convention for arguments is to use underscores and not dashes, please update for every new argument (in other commands too)
```suggestion
    - name: start_date
      description: Date from which to start the event search.
      required: false
      isArray: false
    - name: end_date
      description: Date at which to stop the event search.
```